client/pages/me/me.wxml

```diff
@@ -1,2 +1,50 @@
 <!--pages/me/me.wxml-->
-<text>pages/me/me.wxml</text>
+<view class="container">
+
+
+  <view class="userinfo">   <!-- avatar and username -->
+    <image  class="userinfo-avatar" src="{{personal.avatarUrl}}" mode="cover"></image>
+    <text class="userinfo-nickname">{{personal.nickName}}</text>
+  </view>
+
+
+  <view class="body">   <!-- info -->
+    <view class="info">   <!-- Chinese name -->
+      <text>Chinese name</text>
+      <view class="infoInput">
+        <input wx:if="{{!personal.data.ChineseName}}" placeholder='Tap here to enter' bindinput='chineseNameInput'/>
```

Comment:
I think just `<input placeholder='Tap here to enter' value='{{personal.data.ChineseName}}' 
 bindinput='chineseNameInput'/>` should work, don't need two input fields. Can you try it?